Add further Heartcore URL change info (#6213)
umbraco/UmbracoDocs: umbraco-heartcore/release-notes/core-dependency-upgrade.md

## What to Expect
Your project will be scheduled for upgrade in either July or August. You will receive a notification approximately two weeks prior, with the morning or afternoon of a specific day on which the migration will happen.

The migration will cause a brief partial disruption to service and also contains one breaking change that may affect some sites.

### During Maintenance
You can expect the following services to be affected for approximately one hour during the migration:
* **CMS backoffice** - Users will be unable to log in and author changes to settings, content, and media.
* **Management API** - Applications will be unable to call Management API endpoints. This also impacts the form submission endpoint, which will return HTTP 500-series error responses during the outage window.

### After Maintenence

> After the migration, ***some*** characters in the URL of published content items will be substituted with a different character sequence. This will only affect content items that existed prior to the migration. Additionally it will not affect items until they are next published. Content items that are not re-published will retain their pre-migration URLs.

Reviewer: ```suggestion
After the migration, ***some*** characters in the URL of published content items will be substituted with a different character sequence. This will only affect content items that existed before the migration. Additionally, it will not affect items until they are published next. Content items that are not re-published will retain their pre-migration URLs.
```
